content/references/protocol-reference/transactions/transaction-types/ammbid.md

```diff
@@ -0,0 +1,136 @@
+---
+html: ammbid.html
+parent: transaction-types.html
+blurb: Bid on an Automated Market Maker's auction slot, which grants a discounted fee.
+labels:
+  - AMM
+status: not_enabled
+---
+# AMMBid
+[[Source]](https://github.com/gregtatcam/rippled/blob/amm-core-functionality/src/ripple/app/tx/impl/AMMBid.cpp "Source")
+<!-- TODO: Update source link to merged version when available -->
+
+{% include '_snippets/amm-disclaimer.md' %}
+
+Bid on an Automated Market Maker's (AMM's) auction slot. If this transaction succeeds, you win the bid and can trade against the AMM at a discounted fee until you are outbid or 24 hours have passed. If you are outbid before 24 hours have passed, you are refunded part of the cost of your bid based on how much time remains.
```

Comment:
Hm, I'm not sure how I feel about this suggestion. I like that it's more succinct. My thought behind the original phrasing is that I want to answer the questions, "how do you know if you won?" or "what does it mean for this transaction to be successful?" since there are cases where "success" at the level of executing the transaction doesn't necessarily mean what you might think. (For example, when placing an Offer in the DEX, "success" might mean, "your Offer has been recorded and is waiting for a match", not "you successfully traded currency". Or when canceling a prior Offer, "success" means "there is no longer any such Offer" which can mean "it has been canceled" or "it was already fully consumed" as well as "it never existed because you made a typo".) So I do think it's useful to convey, "if the transaction has a success result code, that _actually_ means you won the bid" and not just, "your bid was received, but it may or may not have been too low to win.")

Reply:
Ah okay. The distinction on what a successful transaction means makes sense to include.